app/representers/space_representer.rb

```diff
@@ -27,5 +27,11 @@ class SpaceRepresenter < Roar::Decorator
     property :admin_id
     property :name
     property :picture
+
+    property :plan_details,
```

Comment:
It only does when it belongs to current_user or is part of their organization.

That's because we need to know if it can be made private.  The plan information tells us the permissions available to it.